### ISSUE TITLE
regarding metatags for fb:

### ISSUE THREAD

**Issue by ds-manish** [vinaygopinath/ng2-meta#36]
it only works when i put:
```
<!-- Facebook integration -->
<meta name="og:url" content="" />
<meta name="og:type" content="" />
<meta name="og:title" content="" />
<meta name="og:description" content="" />
<meta name="og:image" content="" />
```

but if i do this it wont work. fb says it should be `property` not `name`
```
<!-- Facebook integration -->
<meta property="og:url" content="" />
<meta property="og:type" content="" />
<meta property="og:title" content="" />
<meta property="og:description" content="" />
<meta property="og:image" content="" />
```

**Comment by ds-manish** [comment]
this has issue and does not also solve the issue.

Use ` "@nglibs/meta": "^0.4.0-rc.1",` this will work fine and is easy..

**Comment by vinaygopinath** [comment]
Fixed as of ng2-meta v4.0.0 and above
